Sources/protoc-gen-swift/GeneratorOptions.swift

```diff
@@ -33,13 +33,20 @@ class GeneratorOptions {
   enum Visibility {
     case `internal`
     case `public`
+    #if swift(>=5.9)
```

Comment:
Do we really need the `#if` here? We can just leave them out and if the user generates them their code will fail to compile.

Reply:
Yeah okay, good point, this doesn't really safeguard the user from anything - I got rid of them.